crates/plex-api/src/media_container/server/library.rs

```diff
@@ -0,0 +1,496 @@
+    pub last_viewed_at: Option<OffsetDateTime>,
+
+    #[serde(rename = "createdAtTZOffset")]
+    pub created_at_tz_offset: Option<String>,
+    pub created_at_accuracy: Option<String>,
+    #[serde(with = "time::serde::timestamp")]
+    pub added_at: OffsetDateTime,
+    #[serde(default, with = "time::serde::timestamp::option")]
+    pub deleted_at: Option<OffsetDateTime>,
+    #[serde(default, with = "time::serde::timestamp::option")]
+    pub updated_at: Option<OffsetDateTime>,
+    #[serde(default, deserialize_with = "deserialize_option_number_from_string")]
+    pub loudness_analysis_version: Option<u32>,
+    #[serde(default, deserialize_with = "optional_boolish")]
+    pub has_premium_primary_extra: Option<bool>,
+    pub view_offset: Option<u64>,
+    pub chapter_source: Option<ChapterSource>,
+    pub primary_extra_key: Option<String>,
+
+    #[serde(flatten)]
+    pub parent: ParentMetadata,
+    #[serde(flatten)]
+    pub grand_parent: GrandParentMetadata,
+
+    #[serde(default, rename = "Collection")]
+    pub collections: Vec<Tag>,
+    #[serde(default, rename = "Genre")]
+    pub genres: Vec<Tag>,
+    #[serde(default, rename = "Director")]
+    pub directors: Vec<Tag>,
+    #[serde(default, rename = "Writer")]
+    pub writers: Vec<Tag>,
+    #[serde(default, rename = "Country")]
+    pub countries: Vec<Tag>,
+    #[serde(default, rename = "Role")]
+    pub roles: Vec<Role>,
+
+    #[serde(rename = "Media")]
+    pub media: Option<Vec<Media>>,
+}
+
+#[derive(Debug, Deserialize, Clone)]
+#[cfg_attr(feature = "tests_deny_unknown_fields", serde(deny_unknown_fields))]
+#[serde(rename_all = "camelCase")]
+pub struct MetadataMediaContainer {
+    pub key: Option<String>,
+    pub allow_sync: Option<bool>,
+    #[serde(rename = "nocache")]
+    pub no_cache: Option<bool>,
+    pub art: Option<String>,
+    pub banner: Option<String>,
+    #[serde(flatten)]
+    pub parent: ParentMetadata,
+    #[serde(flatten)]
+    pub grand_parent: GrandParentMetadata,
+    #[serde(rename = "librarySectionID")]
+    pub library_section_id: Option<u32>,
+    pub library_section_title: Option<String>,
+    #[serde(rename = "librarySectionUUID")]
+    pub library_section_uuid: Option<String>,
+    pub media_tag_prefix: Option<String>,
+    #[serde(default, with = "time::serde::timestamp::option")]
+    pub media_tag_version: Option<OffsetDateTime>,
+    pub sort_asc: Option<bool>,
+    pub thumb: Option<String>,
+    pub title1: Option<String>,
+    pub title2: Option<String>,
+    pub summary: Option<String>,
+    pub theme: Option<String>,
+    pub view_group: Option<String>,
+    pub view_mode: Option<u32>,
+    #[serde(flatten)]
+    pub media_container: MediaContainer,
+    #[serde(default, rename = "Directory")]
+    pub directories: Vec<Value>,
+    #[serde(default, rename = "Metadata")]
+    pub metadata: Vec<Metadata>,
+}
+
+#[derive(Debug, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub enum PivotType {
```

Comment:
Same as with the previous enum — if you don't have any objections, I would've also preferred having the `Unknown` variant here.